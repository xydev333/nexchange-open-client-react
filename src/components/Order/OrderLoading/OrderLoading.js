import React from 'react';
import { I18n } from 'react-i18next';

const OrderLoading = () => {
  return (<div className="text-center">
      <I18n ns="translations">
		{(t) => (
          <h2>{t('loading')}...</h2>
        )}
      </I18n>
    </div>);
};

export default OrderLoading;
