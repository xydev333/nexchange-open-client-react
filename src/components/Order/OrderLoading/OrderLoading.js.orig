import React from 'react';
import { I18n } from 'react-i18next';

<<<<<<< HEAD:src/components/Order/OrderLoading/OrderLoading.js
const OrderLoading = () => {
  return (
    <div className="text-center">
      <h2>Loading...</h2>
    </div>
  );
=======
const Loading = () => {
    return <div className="text-center">
        <I18n ns="translations">
		{(t) => (
            <h2>{t('loading')}...</h2>
         )}
    	</I18n>
    </div>;
>>>>>>> Start merging i18n integration:src/components/Loading.js
};

export default OrderLoading;
