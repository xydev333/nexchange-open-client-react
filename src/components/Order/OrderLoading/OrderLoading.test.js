import React from 'react';
import { shallow } from 'enzyme';
import OrderLoading from './OrderLoading';

describe('OrderLoading', () => {
  it('renders correctly', () => {
    expect(shallow(<OrderLoading />)).toMatchSnapshot();
  });
});
