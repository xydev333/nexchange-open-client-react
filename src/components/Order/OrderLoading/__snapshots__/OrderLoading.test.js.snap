// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderLoading renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderLoading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        Loading...
</h2>,
      "className": "text-center",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Loading...",
      },
      "ref": null,
      "rendered": "Loading...",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Loading...
</h2>,
        "className": "text-center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Loading...",
        },
        "ref": null,
        "rendered": "Loading...",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
